orderbook/src/database/orders.rs

```diff
@@ -248,7 +250,16 @@ impl OrderStoring for Postgres {
                 WHEN 'sell' THEN sum_sell < sell_amount \
                 WHEN 'buy' THEN sum_buy < buy_amount \
             END) AND \
-            ($7 OR NOT invalidated);";
+            ($7 OR NOT invalidated) \
+        ORDER BY unfiltered.creation_timestamp \
+        LIMIT $8 \
```

Comment:
If i don't remember bad, these formatting syntax was very powerful, and would allow u to use "named placeholders" instead of numbers. I think its the same efficiency and more readable when u have so many params --> hard to spot a mistake.

Reply:
Lol,  don't look at the insertion query (https://github.com/gnosis/gp-v2-services/blob/45dd1d177e00bda303cfa64c7af9680e61a2a49c/orderbook/src/database/orders.rs#L162)

Reply:
What happens if &8 is null? Would that still work?

Reply:
`$8` will either be set or default to `0`, which is equivalent to not having the `OFFSET` clause.

Anyway, this might not look like this at all.
@vkgnosis is working on a different sort of pagination.

This draft is on-hold in the mean time